concordium-sdk/src/main/java/com/concordium/sdk/responses/blocksummary/specialoutcomes/PaydayPoolReward.java

```diff
@@ -0,0 +1,48 @@
+package com.concordium.sdk.responses.blocksummary.specialoutcomes;
+
+import com.concordium.sdk.transactions.CCDAmount;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+
+/**
+ * Payment distributed to a pool or passive delegators.
+ */
+@Getter
+@ToString
+@EqualsAndHashCode(callSuper = false)
+public final class PaydayPoolReward extends SpecialOutcome {
+    /**
+     * The pool owner (passive delegators when 'Null')
+     */
+    private final Long poolOwner;
```

Comment:
I think this should be a UInt64, or maybe introduce an AccountIndex type.

Reply:
I introduced this now. `AccountIndex`.